dev-libs/optix/optix-7.3.0.ebuild

```diff
@@ -0,0 +1,124 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake cuda eapi8-dosym
+
+DESCRIPTION="NVIDIA Ray Tracing Engine"
+HOMEPAGE="https://developer.nvidia.com/optix"
+SRC_URI="NVIDIA-OptiX-SDK-${PV}-linux64-x86_64.sh"
+S="${WORKDIR}"
+
+LICENSE="OptiX-SDK"
+SLOT="0/7"
+KEYWORDS="~amd64"
+IUSE="samples"
+
+RESTRICT="fetch mirror"
+
+RDEPEND="
+	dev-util/nvidia-cuda-toolkit
+	media-libs/freeglut
+	virtual/opengl
+	samples? (
+		dev-libs/libbsd
+		media-libs/openexr:0=
+		media-libs/vulkan-loader
+		sys-libs/zlib
+		virtual/opengl
+		x11-libs/libICE
+		x11-libs/libX11
+		x11-libs/libXau
+		x11-libs/libXcursor
+		x11-libs/libXdmcp
+		x11-libs/libXext
+		x11-libs/libXfixes
+		x11-libs/libXinerama
+		x11-libs/libXrandr
+		x11-libs/libXrender
+		x11-libs/libXxf86vm
+		x11-libs/libxcb:=
```

Comment:
This is a bit awkward but := isn't needed on libxcb, seems glfw dlopen() all those at runtime rather than linking with any of those.

From /var/db/pkg/dev-libs/optix/REQUIRES:
```
x86_64: ld-linux-x86-64.so.2 libIlmImf-2_5.so.25 libc.so.6 libcuda.so.1 libdl.so.2 libgcc_s.so.1 libglfw.so.3 libm.so.6 libnvrtc.so.11.2 libpthread.so.0 librt.so.1 libstdc++.so.6
```
On the other hand, since it's linking with cuda libraries, it needs := on nvidia-cuda-toolkit only with USE=samples

I do kind of wonder the pertinence of depending on nvidia-cuda-toolkit without samples.. I feel like those deps are there from "back when" someone was building samples.